ovos_utils/skills.py

```diff
@@ -26,7 +27,15 @@ def check_class(cls):
     return set(check_class(obj.__class__))
 
 
-def skills_loaded(bus=None):
+def skills_loaded(bus=None) -> Union[bool, str]:
```

Comment:
isnt this always a bool? reply.data["status"] should be a bool too i think

Reply:
No idea, but if it's not always returning a bool I didn't want to risk breaking it/accidentally deprecating

Reply:
did you see it ever not returning a bool? i dont see how that would happen

Reply:
I can't see where the message is ever emitted in OVOS, so I have no idea what value would be coming from the data

Reply:
https://github.com/OpenVoiceOS/ovos-utils/blob/24fdb56438496d2be29905f903c1ab2afac336ca/ovos_utils/process_utils.py#L180

You are correct, this will only ever be a boolean